index.html

```diff
@@ -0,0 +1,160 @@
+          NICOLE GUERRIERO
+          
+          </button>
+        <img src="https://cdn.glitch.com/880db4f1-e059-4f38-b845-ad2307b0e837%2FnicoleE.jpg?v=1607635857708" alt="image of Nicole Guerriero">
+          <p>
+            Makeup and beauty guru with an enormous social media following who was named Ryan Seacrest Best 
+            Beauty Guru of 2013. She runs a blog called Naturally Nicole where she posts tutorials, DIYs, reviews, 
+            and fashion blogs. She worked at a multitude of restaurant jobs, including 5 years working at BJ&apos;s Brewhouse, 
+            before starting YouTube. She also worked as a bartender.
+          </p></li>
+        <li> <button>SOLANGE KNOWLES
+          
+          </button>
+        <img src="https://cdn.glitch.com/880db4f1-e059-4f38-b845-ad2307b0e837%2FsolangeE.jpg?v=1607635859858" alt="image of Solange Knowles">
+          <p>
+            Solange Piaget Knowles is an American singer, songwriter, record producer, performance artist, and actress. 
+            Expressing an interest in music from an early age, Knowles had several temporary stints as a backup dancer 
+            for Destiny&apos;s Child, which featured her elder sister, Beyoncé Knowles, among its members, before signing 
+            with her father Mathew Knowles&apos;s Music World Entertainment label. At age 16, Knowles released her first 
+            studio album Solo Star (2002). Between 2005 and 2007, Knowles had several minor acting roles, including the 
+            direct-to-video Bring It On: All or Nothing (2006) and continued writing music for Beyoncé and for former 
+            Destiny&apos;s Child group members Kelly Rowland and Michelle Williams.
+          </p></li>
+  
+  <form> 
+    <label for="purchase tickets page"> PURCHASE TICKETS</label><br>
+    <p>
+      Each ticket comes with one full day of admission. The cost of each ticket is the same no matter the day of attendance, 
+      or events scheduled that day. <strong>DISCLAIMER:</strong> One ticket will only grant you one day of admission. If you wish to attend multiple dates, 
+      you must purchase multiple tickets. <strong>Each ticket costs $50</strong> and covers one person for a full day of activities. 50% of the proceeds from 
+      ticket sales will go to the Global Fund for Women charity, supporting women worldwide and their pursuit of education and equality. 
+      Learn more at the <a href="https://join.globalfundforwomen.org/a/cye2020a?ms=adwords&gclid=CjwKCAiAq8f-BRBtEiwAGr3DgQYd4C5Lyx_T3hkBVewJ3XnAL8c8J1MqMhgL7cvQHtkBCv4bOoDy1RoC46gQAvD_BwE">
+      Global Fund for Women</a> website.
+    </p>
+    <label for="attending"> SELECT DATES BELOW</label><br>
+    <input type="checkbox" id="thursday"> <label>Thursday May 20th 2021</label><br>
+    <input type="checkbox" id="friday"><label>Friday May 21st 2021</label><br>
+    <input type="checkbox" id="saturday"><label>Saturday May 22nd 2021</label><br>
+    <label>CHECKOUT</label><br>
```

Comment:
You want to make this a heading element, a `p` element or something else -- but not a `label`; `label` only should be used when there is a single `input` that it connects to; they always go hand-in-hand.